lib/action.ts

```diff
@@ -14,9 +14,39 @@ export enum ActionState {
 }
 
 export enum FileFeedEventType {
+  WorkbookAdded = "workbook:added",
+  WorkbookCreated = "workbook:created",
+  WorkbookUpdated = "workbook:updated",
+  WorkbookDeleted = "workbook:deleted",
+  SheetCreated = "sheet:created",
+  SheetUpdated = "sheet:updated",
+  SheetDeleted = "sheet:deleted",
   RecordsCreated = "records:created",
   RecordsUpdated = "records:updated",
   RecordsDeleted = "records:deleted",
+  CommitCreated = "commit:created",
+  CommitUpdated = "commit:updated",
+  CommitDeleted = "commit:deleted",
+  LayerCreated = "layer:created",
+  UploadCompleted = "upload:completed",
+  UploadUpdated = "upload:updated",
+  UploadDeleted = "upload:deleted",
+  FileCreated = "upload:created",
+  FileUpdated = "upload:updated",
+  FileDeleted = "upload:deleted",
+  JobCreated = "job:created",
+  JobReady = "job:ready",
+  JobUpdated = "job:updated",
+  JobCompleted = "job:completed",
+  JobOutcomeAcknowledged = "job:outcome-acknowledged",
+  JobFailed = "job:failed",
+  JobScheduled = "job:scheduled",
+  JobDeleted = "job:deleted",
+  AgentCreated = "agent:created",
+  AgentUpdated = "agent:updated",
+  AgentDeleted = "agent:deleted",
+  SpaceCreated = "space:created",
+  SpaceDeleted = "space:deleted",
```

Comment:
Are these being used? We shouldn't keep a list of statuses, that defeats the purpose of allowing all statuses

Reply:
Removed FileFeedEventType